### ISSUE TITLE
homebridge not showing up in home

### ISSUE THREAD

**Issue by bscuderi13** [nfarina/homebridge#1381]
I've had homebridge succesfully running on a rapsberry pi3 for a while and love it. I had some range issues with the router in some parts of my home although it was great where my pi was located. I decided to upgrade my router 2 days ago and for ease of rejoining all of my devices I had it broadcast the same SSID and Passkey. All my devices joined without problems. I did however notice that siri control started to become flaky. Im not positive it was the router change that caused it but sometimes it wouldn't hear back from my devices it went from 100 percent reliability to less than 50. The pi has great signal and is in the same room as the router. Today I couldn't get it to wok at all and in the home app it showed all my devices as not responding. I ssh into the pi and everything is running great without error. I decided to delete the home and try and repair to no avail it wont show up. I even tried removing the persist and accesories directories, changing the name, mac adress, port, and pin etc. I even tried disabling it on the pi and doing a fresh install on an odroid xu4 I have hardwired to the router and all the same steps. Homebridge is running fine on the odroid as well and it wont show up in the home app. I'm at a loss with what to do or what could have caused this. The router was changed from a netgear 1900 or something like that to an asus 3200 or something like that if that helps.

**Comment by bscuderi13** [comment]
update I also thought maybe it was an ipv6 issue through some of my reading so also tried disabling ipv6 on the device and in avahi still nothing. Fing network scanner cant find the port either. Im out of ideas.

**Comment by bscuderi13** [comment]
I was able to resolve this on a hunch I thought it all was getting blocked somehow since it was all showing up and running. I disabled the router firewall then changed username mac address and port number as well as deleting the persist and accesories just like before. I was able to pair and control devices again. I'm not sure why the new routers firewall was blocking it. There was no option to change or allow specific services from being blocked or allowed that I could see. Hopefully anyone with an ASUS router can help me cause I'm not sure why it was blocking anything especially internal traffic like homebridge? Id definitely rather turn my firewall back on. A web search isnt indicating anyone else having firewall problems with homebridge.

**Comment by githubberboy** [comment]
I can't help you about the firewall problem, but I had the same problem with the homebridge, that no longer displayed after removed from the Apple Home application. I solved it by changing (every time) the "username" parameter in ".homebridge / config.json", simply increasing it by 1.

**Comment by robboz4** [comment]
I hit this problem last night after editing the config.json file. Everything was working fine and I was adding devices. Then nothing. I rebooted the pi3 and still nothing. So I added a 1 to the username in the config file and it works. Thanks for the work around, but why does the username need to be increased to solve the problem. Did the old username expire after several tries?

**Comment by bscuderi13** [comment]
I actually finally solved this with being able to run the firewall. I guess I got lucky with the firewall off cause it was still intermittent. I noticed that none of my apple services were running or showing including airplay airplay etc. that helped me hone it in a web search that pointed me to a setting in my asus router under wireless then professional. Enable igmp snooping. I've enabled this and re enabled my firewall and everything is now working homebridge has been solid for a week same with airplay. I'll close this out but hopefully it helps anyone on my boat. I wished asus labeled that setting in bold and said in plain English it may block apple services to be disabled which it is by default.

**Comment by robboz4** [comment]
It looks like two issues: one with your router set up and the other with adding one to the user I'd. I made no changes to my router. I was Justin's trying to add devices to the config.json file. Perhaps we should open a new issue regarding this.
